code/game/objects/effects/decals/Cleanable/humans.dm

```diff
@@ -133,12 +138,26 @@ var/global/list/image/splatter_cache=list()
 	icon_state = "1"
 	random_icon_states = list("1","2","3","4","5")
 	amount = 0
-	var/list/drips = list()
+	var/drips = 1
+	bloodiness = 0
+
+/obj/effect/decal/cleanable/blood/drip/can_bloodcrawl_in()
+	return FALSE
```

Comment:
On tg they done like them using drips I think but not opposed myself if you really want it